### ISSUE TITLE
[Introspection] Space is not accepted as character in column name

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1080]
Error Message:
```
error: Unexpected token. Expected one of: field type.
  -->  schema.prisma:36
   | 
35 |   length_2             Float?
36 |   nomi l_size_1        Float?
   |                             ^ Unexpected token.
37 |   nomi l_size_2        Float?
   | 
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-b3d256966d10085feaa149b5fecc64d1)

Model:
```
model comp_adaptor {
  adaptor_angle        Int?
  component_id         components?
  component_type_id    type_component?
  connection_type_id_1 type_connection? @relation("comp_adaptor_connection_type_id_1Totype_connection")
  connection_type_id_2 type_connection? @relation("comp_adaptor_connection_type_id_2Totype_connection")
  end_form_id_1        type_end_form?   @relation("comp_adaptor_end_form_id_1Totype_end_form")
  end_form_id_2        type_end_form?   @relation("comp_adaptor_end_form_id_2Totype_end_form")
  hex_size             Float?
  length_1             Float?
  length_2             Float?
  nomi l_size_1        Float?
  nomi l_size_2        Float?
  orientation          String?
  overall_length       Float?
  thread_pitch_1       Float?
  thread_pitch_2       Float?
  thread_size_1        Float?
  thread_size_2        Float?
  unique_feature       String?
  weight               Float?
}
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-5e953d24fce61401f5f1c6317fba191bR27-R48)

original DB:
```
CREATE TABLE `comp_adaptor` (
	`component_id` VARCHAR(255) NULL DEFAULT NULL,
	`component_type_id` VARCHAR(255) NULL DEFAULT NULL,
	`adaptor_angle` INT(11) NULL DEFAULT NULL,
	`overall_length` DECIMAL(65,30) NULL DEFAULT NULL,
	`end_form_id_1` VARCHAR(255) NULL DEFAULT NULL,
	`connection_type_id_1` VARCHAR(255) NULL DEFAULT NULL,
	`length_1` DECIMAL(65,30) NULL DEFAULT NULL,
	`thread_size_1` DECIMAL(65,30) NULL DEFAULT NULL,
	`thread_pitch_1` DECIMAL(65,30) NULL DEFAULT NULL,
	`nomi l_size_1` DECIMAL(65,30) NULL DEFAULT NULL,
	`end_form_id_2` VARCHAR(255) NULL DEFAULT NULL,
	`connection_type_id_2` VARCHAR(255) NULL DEFAULT NULL,
	`length_2` DECIMAL(65,30) NULL DEFAULT NULL,
	`thread_size_2` DECIMAL(65,30) NULL DEFAULT NULL,
	`thread_pitch_2` DECIMAL(65,30) NULL DEFAULT NULL,
	`nomi l_size_2` DECIMAL(65,30) NULL DEFAULT NULL,
	`hex_size` DECIMAL(65,30) NULL DEFAULT NULL,
	`unique_feature` VARCHAR(255) NULL DEFAULT NULL,
	`orientation` VARCHAR(255) NULL DEFAULT NULL,
	`weight` DECIMAL(65,30) NULL DEFAULT NULL,
	INDEX `comp_adaptor_post_component_id_fkey` (`component_id`),
	INDEX `comp_adaptor_post_component_type_id_fkey` (`component_type_id`),
	INDEX `comp_adaptor_post_end_form_id_2_fkey` (`end_form_id_2`),
	INDEX `comp_adaptor_post_end_form_id_1_fkey` (`end_form_id_1`),
	INDEX `comp_adaptor_post_connection_type_id_1_fkey` (`connection_type_id_1`),
	INDEX `comp_adaptor_post_connection_type_id_2_fkey` (`connection_type_id_2`),
	CONSTRAINT `comp_adaptor_post_component_id_fkey` FOREIGN KEY (`component_id`) REFERENCES `components` (`component_id`) ON UPDATE CASCADE ON DELETE CASCADE,
	CONSTRAINT `comp_adaptor_post_component_type_id_fkey` FOREIGN KEY (`component_type_id`) REFERENCES `type_component` (`component_type_id`) ON UPDATE CASCADE ON DELETE CASCADE,
	CONSTRAINT `comp_adaptor_post_connection_type_id_1_fkey` FOREIGN KEY (`connection_type_id_1`) REFERENCES `type_connection` (`connection_type_id`) ON UPDATE CASCADE ON DELETE CASCADE,
	CONSTRAINT `comp_adaptor_post_connection_type_id_2_fkey` FOREIGN KEY (`connection_type_id_2`) REFERENCES `type_connection` (`connection_type_id`) ON UPDATE CASCADE ON DELETE CASCADE,
	CONSTRAINT `comp_adaptor_post_end_form_id_1_fkey` FOREIGN KEY (`end_form_id_1`) REFERENCES `type_end_form` (`end_form_id`) ON UPDATE CASCADE ON DELETE CASCADE,
	CONSTRAINT `comp_adaptor_post_end_form_id_2_fkey` FOREIGN KEY (`end_form_id_2`) REFERENCES `type_end_form` (`end_form_id`) ON UPDATE CASCADE ON DELETE CASCADE
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB
ROW_FORMAT=COMPACT
;

```

![image](https://user-images.githubusercontent.com/183673/70396511-05f29300-1a0a-11ea-8884-ab671634a297.png)

**Comment by pantharshit00** [comment]
I can confirm this bug. A temporary workaround here would be to add the @map directive to those fields and changing the identifier to a valid name.

**Comment by janpio** [comment]
fixed via #1139
